aspnetcore/signalr/configuration.md

```diff
@@ -1808,6 +1815,7 @@ Additional options can be configured in the `WithUrl` (`withUrl` in JavaScript)
 | JavaScript Option | Default Value | Description |
 | ----------------- | ------------- | ----------- |
 | `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |
+| `transport` | `null` | An <xref:Microsoft.AspNetCore.Http.Connections.HttpTransportType> value specifying the transport to use for the connection. |
```

Comment:
@BrennanConroy This is for `::: moniker range="< aspnetcore-2.2"`, so it goes back before 2.1, but this has been on IHttpConnectionOptions longer than anything else in this table.

https://github.com/dotnet/aspnetcore/commit/d169b96d2d794d52ae4c613df0854e2525314500#diff-5fb15f68b85f362298c3477405ec237742d208c5eaca152615a4c814a672f6b1R4-R7

Reply:
Ok? I feel like I'm missing something. Is there a problem?

Reply:
You asked for me to add this for "2.1+". I don't want to have to add a new moniker range. Sounds like we're in agreement that I don't have to.

Reply:
These docs don't exist for <2.1 so we're fine :)